lib/hexpm/emails/emails.ex

```diff
@@ -65,6 +65,15 @@ defmodule Hexpm.Emails do
     |> render("organization_invite.html")
   end
 
+  def package_published(user, package) do
+    email()
+    |> email_to(Enum.map(user.emails, fn email -> email.email end))
+    |> subject("Hex.pm - Package #{package.name} published")
```

Comment:
can't think of a use case where this would be needed but I'd also include the version that was just released, seems more complete that way. wdyt?